Acrolinx.Net/Acrolinx.Net/AcrolinxEndpoint.cs

```diff
@@ -107,7 +107,8 @@ private string GetAbsoluteRequestUrl(string apiPath)
             {
                 return apiPath;
             }
-            return acrolinxUrl + "/api/v1/" + apiPath;
+
+            return acrolinxUrl.TrimEnd('/') + "/api/v1/" + apiPath;
```

Comment:
Can you add unit tests for this specific TrimEnd change. 
covering at-least following scenarios:

https://tenant.acrolinx-cloud.net/
https://tenant.acrolinx-cloud.net//
https://tenant.acrolinx-cloud.net

Reply:
@abhijeetnarvekar  I added a unit test for the url variations!